examples/add-receiver-creator/rendered_manifests/configmap-cluster-receiver.yaml

```diff
@@ -66,6 +66,10 @@ data:
         - system
         override: true
         timeout: 10s
+      resourcedetection/k8s_cluster_name:
+        detectors: null
+        override: true
+        timeout: 10s
```

Comment:
Can we avoid introducing this processor if it's not need?

Reply:
There are checks to only include the processor in the pipeline if it is detecting the GKE/EKS cluster name, but I agree, there's no point in defining a processor if it's not being used. This processor will now only be defined if it's going to be used (if cluster name is optional).